shipstation/api.py

```diff
@@ -398,3 +398,30 @@ def post(self, endpoint='', data=None):
         )
         if self.debug:
             pprint.PrettyPrinter(indent=4).pprint(r.json())
+    
+    def download(self):
```

Comment:
It looks like your CI tests are failing because this method isn't indented. 

Seems like this is still a work in progress, is it possible you forgot to push some commits to your repo before opening this PR?